src/org/usfirst/frc/team2706/robot/commands/autonomous/auto2018/automodes/CenterStartLeftSwitch.java

```diff
@@ -0,0 +1,26 @@
+package org.usfirst.frc.team2706.robot.commands.autonomous.auto2018.automodes;
+
+import org.usfirst.frc.team2706.robot.commands.EjectCube;
+import org.usfirst.frc.team2706.robot.commands.SetLiftHeight;
+import org.usfirst.frc.team2706.robot.commands.autonomous.core.RotateDriveWithGyro;
+import org.usfirst.frc.team2706.robot.commands.autonomous.core.StraightDriveWithEncoders;
+
+import edu.wpi.first.wpilibj.command.CommandGroup;
+
+public class CenterStartLeftSwitch extends CommandGroup {
+    
+    public CenterStartLeftSwitch() {
+       // CurveDrive c = new CurveDrive(76 / 12.0, 94 / 12.0, 0, 0.6, true, 0.25, "CurveToSfawfawfwitch");
+        //this.addSequential(c);
+        this.addSequential(new StraightDriveWithEncoders(0.8, 0.96, 1, 1, this + ".awayFromWall"),1);
+        this.addSequential(new RotateDriveWithGyro(0.6, -36, 1, 2, this + ".faceSwitch"),2);
```

Comment:
Same comment as above about the constants.... Maybe we can figure this out next year....